appconfig/passwords/accesslist.txt

```diff
@@ -15,4 +15,5 @@ chainedtp:pass
 sortslave:pass
 metrics:pass
 vwapsub:pass
+dqe:pass
```

Comment:
think you need a `dqe.txt` file as well; `accesslist.txt` defines what usernames & passwords are _accepted_ for incoming connections, `{proctype}.txt` defines what username & password is _used_ for outgoing connections